admin-manual.md

```diff
@@ -0,0 +1,187 @@
+# Running a MISO Instance
+MISO requires some configuration directly in the source code. While we plan to
+change this over time, running an instance of MISO will require building and
+deploying a fork of the code base with the customisations.
+
+# Creating a Fork and Setting Configuration
+Use the GitHub interface or a private instance to create a forked repository.
+
+Proceed to set up a build environment, then edit
+`miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes.
+The naming schemes will determine how MISO checks if object names (sample,
+library, etc) are valid. If you do not want to use one of the supplied ones
+(TGAC's standard, OICR's standard, or no checks), it is likely you will have to
+write one specific to your organisation.
+
+# Setting Up the Build Environment
+One or machines should be set up to build MISO. This should probably not be on
+the production server. A typical Linux, Windows, or MacOS system will work.
```

Comment:
Why shouldn't it be on the production server? Also, are we sure that MISO works on Windows? Are all of the path separators right when accessing local-ish files, for instance on the notification server?